app/packages/core/src/components/Sidebar/Entries/Draggable.tsx

```diff
@@ -38,15 +41,21 @@ const Draggable: React.FC<
         : "grab"
       : "pointer",
   });
+  const dataCyKey = entryKey
+    ?.split(",")?.[1]
+    ?.replace(/["]/g, "")
+    ?.replace("]", "");
 
   return (
     <>
       <animated.div
+        data-draggable={!disableDrag && trigger && !isReadOnly}
```

Comment:
nit: since this is evaluated in every render, consider wrapping this in a `useMemo()`

```
const isDraggable = useMemo(() => !disableDrag && trigger && !isReadOnly, [disableDrag, trigger, isReadOnly])
```